FiveCalls/FiveCalls/Colors.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Colors.swift
+//  FiveCalls
+//
+//  Created by Nick O'Neill on 8/25/23.
+//  Copyright © 2023 5calls. All rights reserved.
+//
+
+import SwiftUI
+
+extension Color {
+    static let FiveCallsLightBG = Color(red: 0.85, green: 0.85, blue: 0.85)
+    static let FiveCallsDarkBlue = Color("darkBlue")
```

Comment:
I'm actually not convinced this will work, but see my comment [here](https://github.com/5calls/ios/pull/337#discussion_r1327418576) for a better alternative.